src/pluto/format/jpeg.cr

```diff
@@ -1,6 +1,10 @@
 module Pluto::Format::JPEG
   macro included
-    def self.from_jpeg(image_data : String) : Image
+    def self.open_jpeg(filename : String) : self
+      from_jpeg(File.read(filename))
+    end
```

Comment:
Let's remove such methods as well because they add extra complexity for what can be easily done manually.